tests/suites/test_suite_x509parse.data

```diff
@@ -1498,29 +1498,31 @@ X509 Certificate ASN1 (TBSCertificate, invalid SubjectPublicKeyinfo, check faile
 depends_on:MBEDTLS_RSA_C:MBEDTLS_SHA256_C
 x509parse_crt:"308196308180a0030201008204deadbeef300d06092a864886f70d01010b0500300c310a30080600130454657374301c170c303930313031303030303030170c303931323331323335393539300c310a30080600130454657374302a300d06092A864886F70D010101050003190030160210fffffffffffffffffffffffffffffffe0202ffff300d06092a864886f70d01010b0500030200ff":"":MBEDTLS_ERR_PK_INVALID_PUBKEY
 
-X509 Certificate ASN1 (TBSCertificate v3, Optional UIDs, Extensions not present)
+# We don't expect MBEDTLS_ERR_ASN1_UNEXPECTED_TAG here because the IssuerID is optional;
+# the stack is expected to ignore it and will hence end up with a length mismatch.
+X509 Certificate ASN1 (TBSCertificate, invalid IssuerID, invalid tag)
```

Comment:
You're right, the documentation is outdated. It would make sense in v2 Certificates which don't contain extensions, but it doesn't make sense here. I pushed a commit to fix this.